flink-clients/src/test/java/org/apache/flink/client/program/rest/RestClusterClientTest.java

```diff
@@ -542,6 +549,68 @@ public void testListJobs() throws Exception {
 		}
 	}
 
+	@Test
+	public void testGetAccumulators() throws Exception {
+		TestAccumulatorHandlers accumulatorHandlers = new TestAccumulatorHandlers();
+		TestAccumulatorHandlers.TestAccumulatorHandler accumulatorHandler = accumulatorHandlers.new TestAccumulatorHandler();
+
+		try (TestRestServerEndpoint ignored = createRestServerEndpoint(accumulatorHandler)){
+
+			JobID id = new JobID();
+
+			{
+				Map<String, Object> accumulators = restClusterClient.getAccumulators(id);
+				assertNotNull(accumulators);
+				assertEquals(1, accumulators.size());
+
+				assertEquals(true, accumulators.containsKey("testKey"));
+				assertEquals("testValue", accumulators.get("testKey").toString());
+			}
+		}
+	}
+
+	private class TestAccumulatorHandlers  {
+
+		private class TestAccumulatorHandler extends TestHandler<EmptyRequestBody, JobAccumulatorsInfo, JobAccumulatorsMessageParameters> {
+
+			public TestAccumulatorHandler() {
+				super(JobAccumulatorsHeaders.getInstance());
+			}
+
+			@Override
+			protected CompletableFuture<JobAccumulatorsInfo> handleRequest(@Nonnull HandlerRequest<EmptyRequestBody,
+				JobAccumulatorsMessageParameters> request, @Nonnull DispatcherGateway gateway) throws RestHandlerException {
+				JobAccumulatorsInfo accumulatorsInfo;
+				List<Boolean> queryParams = request.getQueryParameter(AccumulatorsIncludeSerializedValueQueryParameter.class);
+
+				boolean includeSerializedValue = false;
+				if (!queryParams.isEmpty()) {
+					includeSerializedValue = queryParams.get(0);
+				}
+
+				List<JobAccumulatorsInfo.UserTaskAccumulator> userTaskAccumulators = new ArrayList<JobAccumulatorsInfo.UserTaskAccumulator>() {{
```

Comment:
Instead of creating a new anonymous class can we simply create an `ArrayList` and then add the element (without the initializer block)?